packages/vscode/src/plugins/studio/index.ts

```diff
@@ -0,0 +1,100 @@
+import * as cp from 'child_process'
+import * as vscode from 'vscode'
+import { getResourceURI } from '../../util'
+import { PrismaVSCodePlugin } from '../types'
+
+let studioProcess: cp.ChildProcessWithoutNullStreams
+
+const plugin: PrismaVSCodePlugin = {
+  name: 'studio',
+  enabled: () => true,
+  commands: [
+    {
+      id: 'prisma.plugin.studio.kill',
+      action: () => {
+        if (studioProcess) {
```

Comment:
There should be a log output here in case the command is run, even though studioProcess is undefined.